flashcard-maker/src/lib/lib_node.js

```diff
@@ -0,0 +1,24 @@
+class LibNode {
+    fs = require('fs');
+    writeDataFile(data) {
```

Comment:
Just one question: How do we capture the user input? Do we get the input from a `textarea` element or a text file?

Reply:
I was under the impression that we were going to get the input from a textarea, then save it into a file... if we're just going to save everything into a db, then I guess it's not necessary? (initially, I actually didn't really think making a Node class and extending it was even necessary because of this haha)

Reply:
I would suggest that we can comment out those code for now and then update it later.

Reply:
sounds good 👍